docs/ReferenceResolution.md

```diff
@@ -100,16 +100,65 @@ is changing to `Microsoft.AspNetCore.BetterThanOrange`, you would need to make t
   </ItemGroup>
 ```
 
-## Updating dependencies manually
-
-If the `dotnet-maestro` bot has not correctly updated the dependencies, it may be worthwhile running `darc` manually:
-
-1. Install `darc` as described in <https://github.com/dotnet/arcade/blob/master/Documentation/Darc.md#setting-up-your-darc-client>
-2. Run `darc update-dependencies --channel '.NET Core 3.1 Release'`
-   * Use `'trigger-subscriptions'` to prod the bot to create a PR if you do not want to make local changes
-   * Use `'.NET 3 Eng''` to update dependencies from dotnet/arcade
-   * Use `'.NET Eng - Latest'` to update dependencies from dotnet/arcade in the `master` branch
-   * Use `'VS Master'` to update dependencies from dotnet/roslyn in the `master` branch
-   * Use `'.NET 5 Dev'` to update dependencies from dotnet/efcore or dotnet/runtime in the `master` branch
-3. `git diff` to confirm the tool's changes
-4. Proceed with usual commit and PR
+## A darc cheatsheet
+
+`darc` is a command-line tool that is used for dependency management in the dotnet ecosystem of repos. `darc` can be installed using the `darc-init` scripts located inside the `eng/common` directory. Once `darc` is installed, you'll need to set up the appropriate access tokens as outlined [in the official Darc docs](https://github.com/dotnet/arcade/blob/master/Documentation/Darc.md#setting-up-your-darc-client).
+
+Once `darc` is installed and set-up, it can be used to modify the subscriptions and dependencies in a project.
+
+**Getting the list of subscriptions in a repo**
+
+Subscriptions are objects that define the ecosystem repos we are listening for updates to, the frequency we are looking for updates, and more.
+
+```
```

Comment:
nit: VS Code will complain about not specifying the type of code block. This is pretty generic so `bash` or `ps1` would both work. If you do this, remove the leading `$`s and do it for all the scripting you're adding.